library/Icingadb/Model/Behavior/ReRoute.php

```diff
@@ -12,6 +12,13 @@ class ReRoute implements RewriteFilterBehavior, RewritePathBehavior
 {
     protected $routes;
 
+    /**
+     * Special tables which require an additional step to apply filters with service group name
```

Comment:
That's not very informative. The name of the constant is neither.

Suggestion:

**Description**: `Tables with mixed object type entries for which servicegroup filters need to be resolved in multiple steps`
**Constant**: `MIXED_TYPE_RELATIONS`